CHANGELOG.md

```diff
@@ -3,7 +3,8 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-___NULL___
+* Azure Function types for Blob, Service Bus, Event Hub, IoT Hub, and Cosmos DB triggers
+* BREAKING: Types related to ServiceBus Topic subscriptions: TopicContext, TopicHostSettings, and TopicCallback were replaced by more general types ServiceBusContext, ServiceBusHostSettings, and ServiceBusCallback to support queue-based scenarios.
```

Comment:
any way to avoid the break?

Reply:
Yes, copy everything twice - keep `TopicXyz` and add `QueueXyz`, or use `TopicXyz` for queues which would look weird but will work.
Those 3 changed types aren't used explicitly in most cases, so I'm hoping it's not a big issue for users?

Reply:
Yeah, we added this recently enough that i'm ok with breaking.  Basically, it allows us to move fast, but quickly iterate if we want.  As things bake and become more solid we should be more careful about not breaking. but i think it's ok here as these are veyr very new.